client/header/activity-panel/highlight-tooltip/style.scss

```diff
@@ -0,0 +1,42 @@
+.highlight-tooltip__container {
+	position: absolute;
+	width: 0;
+	height: 0;
+	&.highlight-tooltip__show {
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+	}
+}
+.highlight-tooltip__portal {
+	width: 100%;
+	height: 100%;
+	position: relative;
+
+	.highlight-tooltip__overlay {
+		position: fixed;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+		background-color: rgba($black, 0.35);
+		z-index: z-index('.components-modal__screen-overlay');
+
+		@include edit-post__fade-in-animation();
```

Comment:
TIL!

I'm not able to see the `0.2s` fade in. 😛

Reply:
Yeah I copied that from the [Modal css](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/modal/style.scss#L12).
I can't seem to really see it either 🙈